.github/workflows/pages.yaml

```diff
@@ -0,0 +1,41 @@
+name: Deploy Github Pages
+on:
+  push:
+    branches: [main]
+      
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+concurrency:
+  group: "pages"
+  cancel-in-progress: true
+
+jobs:
+  deploy-pages:
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+      
+      - name: Generate inventory
+        run: |
+          python rules_inventory/scripts/rules_overview_generator.py --rules_file=rules/falco_rules.yaml > docs/overview.md
```

Comment:
We generate the overview page dynamically.

Reply:
In a follow up PR we could find a new home for the python script? For example directly more hidden in the `.github`? Let me know and I was gonna update the script now that most rules are already tagged.

Reply:
We can do that for sure!